src/test/java/com/codicesoftware/plugins/hudson/PlasticSCMTest.java

```diff
@@ -14,10 +14,9 @@
 public class PlasticSCMTest {
 
     @Rule
-    public JenkinsRule jenkins = new JenkinsRule();
+    public JenkinsRule rule = new JenkinsRule();
 
-    @Test
-    @Ignore
+    //@Test
```

Comment:
We should remove the test completely rather than removing the `@Test` annotation, right? Otherwise I prefer to have it marked as test but ignored, as it was before.

Reply:
If the test is ignored then running Maven test phase will report this test as skipped, giving a warning. This looks bad. The test was ignored because depending on the test environment it will have Plastic CLI `cm` installed or not. We are unsure of this and this test can not be trusted. I kept this test for my own checks. Will remove it as it is not generic enough to be properly useful.